packages/core/src/lib/interfaces.ts

```diff
@@ -154,33 +162,36 @@ export interface Frontend<O extends Options = Options> {
    * the returned Promise has resolved before issuing methods such as
    * {@link Frontend.captureException} or {@link Frontend.captureBreadcrumb}.
    *
-   * @returns A Promise that resolves when installation has finished.
+   * @returns If the installation was the successful or not.
    */
-  install(): Promise<boolean>;
+  install(): boolean;
 
   /**
    * Captures an exception event and sends it to Sentry.
    *
    * @param exception An exception-like object.
-   * @returns A Promise that resolves when the exception has been sent.
+   * TODO
+   * @returns The created event id.
    */
-  captureException(exception: any): Promise<void>;
+  captureException(exception: any, scope: Scope): Promise<void>;
 
   /**
    * Captures a message event and sends it to Sentry.
    *
    * @param message The message to send to Sentry.
-   * @returns A Promise that resolves when the message has been sent.
+   * TODO
+   * @returns The created event id.
    */
-  captureMessage(message: string): Promise<void>;
+  captureMessage(message: string, scope: Scope): Promise<void>;
 
   /**
    * Captures a manually created event and sends it to Sentry.
    *
    * @param event The event to send to Sentry.
-   * @returns A Promise that resolves when the event has been sent.
+   * TODO
+   * @returns The created event id.
    */
-  captureEvent(event: SentryEvent): Promise<void>;
+  captureEvent(event: SentryEvent, scope: Scope): Promise<void>;
```

Comment:
Shouldn't all these return an event ID based on recent discussions? Also, the return type is still `async`, however `addBreadcrumb` is not, whereas both **might** be asynchronous internally (due to specific SDK implementations).